src/codeu/chat/common/ServerInfo.java

```diff
@@ -12,7 +12,7 @@ public ServerInfo(){
       try {
         temp = Uuid.parse(SERVER_VERSION);
       } catch (IOException e) {
-          System.out.println("Fatal Error: Cannot parse the server infomation. Line 13, ServerInfo.java");
+          e.printStackTrace("Fatal Error: Cannot parse the server infomation. All variables in this instance will be null this time.");
```

Comment:
1. What makes this error fatal?
2. Same syntax problem w/printStackTrace, different lines.
3. Second error sentence is unnecessary because nothing guaranteeable happens in the program after this message and its syntax is printed.